docs/writing-rules/private-rules.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
What do you think about placing this file inside the `docs/semgrep-app` folder? I see the argument for both, though I lean more towards placing it with other app-specific items.